src/bb-modules/Servicesolusvm/SolusVM.php

```diff
@@ -1,5 +1,19 @@
 <?php
 
+/**
+ * FOSSBilling.
+ *
+ * @copyright FOSSBilling (https://www.fossbilling.org)
+ * @license   Apache-2.0
+ *
+ * Copyright FOSSBilling 2022
+ * This software may contain code previously used in the BoxBilling project.
+ * Copyright BoxBilling, Inc 2011-2021
+ *
+ * This source file is subject to the Apache-2.0 License that is bundled
+ * with this source code in the file LICENSE
+ */
+
```

Comment:
This one too. Maybe we shouldn't automatically apply headers?

Reply:
> Maybe we shouldn't automatically apply headers?

I think it's fine, I just added exclusions for those folders & most things we add that isn't our code are through composer these days so this won't even effect that